docs/Installing.asciidoc

```diff
@@ -99,10 +99,50 @@ You can do that in +/etc/openqa/openqa.ini+:
 httpsonly = 0
 --------------------------------------------------------------------------------
 
+=== Database engine (PostgreSQL)
+
+To setup openQA you will need a PostgreSQL database available to the user
+running openQA (geekotest).
+
+You need to create a database (openqa) and a dedicated user account (geekotest)
+with full access to it. To configure access to the chosen database in openQA,
+edit +/etc/openqa/database.ini+ and change the settings in the +[production]+
+section.
+
+[source.ini]
```

Comment:
I guess that should be `[source,sh]` instead. The following is not a file content but commands to execute